### DISCUSSION TITLE
Bundle install error

### DISCUSSION THREAD

**Discussion by haoliu-90** [alshedivat/al-folio#1082] [Q&A]
Hi, I am a new user of github pages. After clone the repo to my local machine and ran the command "bundle install". I have the following error. Could you please help me with this? Thank you in advance. 

Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

current directory:
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/libv8-node-16.10.0.0/ext/libv8-node
C:/Ruby31-x64/bin/ruby.exe -I C:/Ruby31-x64/lib/ruby/3.1.0 extconf.rb
creating Makefile
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/libv8-node-16.10.0.0/ext/libv8-node/builder.rb:12:in
`build_libv8!': failed to download node 16.10.0 (Libv8::Node::BuilderError)
from
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/libv8-node-16.10.0.0/ext/libv8-node/location.rb:30:in
`install!'
        from extconf.rb:9:in `<main>'
==== in
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/libv8-node-16.10.0.0/ext/libv8-node
==== running
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/libv8-node-16.10.0.0/libexec/download-node

extconf failed, exit code 1

Gem files will remain installed in
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/libv8-node-16.10.0.0 for inspection.
Results logged to
C:/Ruby31-x64/lib/ruby/gems/3.1.0/extensions/x64-mingw-ucrt/3.1.0/libv8-node-16.10.0.0/gem_make.out

  C:/Ruby31-x64/lib/ruby/3.1.0/rubygems/ext/builder.rb:102:in `run'
  C:/Ruby31-x64/lib/ruby/3.1.0/rubygems/ext/ext_conf_builder.rb:28:in `build'
  C:/Ruby31-x64/lib/ruby/3.1.0/rubygems/ext/builder.rb:171:in `build_extension'
C:/Ruby31-x64/lib/ruby/3.1.0/rubygems/ext/builder.rb:205:in `block in
build_extensions'
  C:/Ruby31-x64/lib/ruby/3.1.0/rubygems/ext/builder.rb:202:in `each'
  C:/Ruby31-x64/lib/ruby/3.1.0/rubygems/ext/builder.rb:202:in `build_extensions'
  C:/Ruby31-x64/lib/ruby/3.1.0/rubygems/installer.rb:843:in `build_extensions'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/rubygems_gem_installer.rb:72:in
`build_extensions'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/rubygems_gem_installer.rb:28:in
`install'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/source/rubygems.rb:207:in
`install'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/installer/gem_installer.rb:54:in
`install'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/installer/gem_installer.rb:16:in
`install_from_spec'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/installer/parallel_installer.rb:186:in
`do_install'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/installer/parallel_installer.rb:177:in
`block in worker_pool'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/worker.rb:62:in
`apply_func'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/worker.rb:57:in
`block in process_queue'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/worker.rb:54:in
`loop'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/worker.rb:54:in
`process_queue'
C:/Ruby31-x64/lib/ruby/gems/3.1.0/gems/bundler-2.3.25/lib/bundler/worker.rb:91:in
`block (2 levels) in create_threads'

An error occurred while installing libv8-node (16.10.0.0), and Bundler cannot
continue.

In Gemfile:
  mini_racer was resolved to 0.6.3, which depends on
    libv8-node

**Comment by george-gca** [comment]
Addressed as #1520
